### ISSUE TITLE
FIrefox theme only changing on restart

### ISSUE THREAD

**Issue by embie27** [oskarsh/Yin-Yang#117]
As far as I can see, the firefox extension only applies scheduled theme switches, not manual ones.

Did I miss the code section or is this the expected behaviour?

**Comment by l0drex** [comment]
Yes, manual themes get only applied when starting the browser. The plugin loads the settings of yin yang and then the connection is never used again.
As far as I know, there is no way to send a message on initiative from the python script, so I didn't implement that.

**Comment by bphd** [comment]
You can use a (WayLand WM/DE)[https://wiki.archlinux.org/title/Wayland#Compositors] and use (FireFox on WayLand)[https://aur.archlinux.org/packages/firefox-wayland-mode-hook] with (default automatic theme)[]

If all of that is rightly installed and activated, it should follow the system theme automatically, reading DE/WM theme. Without any restart

**Comment by l0drex** [comment]
The default theme is working fine on x11 for me (by default). 
The plugin is there for setting custom themes, since Firefox can't sync those with the system theme.

**Comment by Dieterbe** [comment]
>  The plugin is there for setting custom themes, since Firefox can't sync those with the system theme.

interesting. so if we're happy with the default light and default dark theme in firefox, we don't need to enable the plugin in yin-yang (we only need to install the add-on in firefox?) - that's interesting, and perhaps something that can be clarified in the yin-yang plugin UI

**Comment by l0drex** [comment]
You don't even need the plugin in Firefox, just choose the system theme and thats it 😄
